### ISSUE TITLE
Enable fieldmap jacobians by default only for PEPolar fieldmaps

### ISSUE THREAD

**Issue by effigies** [nipreps/fmriprep#3367]
With 9 months of experience behind us, IMO `--ignore fmap-jacobian` should always be used if the fieldmap is not PEPolar, but it is critical for at least some PEPolar cases.

I think we should disable it by default, which means we'll need to add a flag to reenable it. We could either do `--fmap-jacobian`, `--force-fmap-jacobian` or add a `--force` similar to `--ignore`, and permit `fmap-jacobian` as a value.

I think we should either go with `--force fmap-jacobian` or deprecate `--ignore fmap-jacobian` and use `--[no-]fmap-jacobian`.

If we go with `--force`, we could transition some other flags away from the `--[no-]<whatever>` form.

**Comment by tsalo** [comment]
This might be a dumb question, but isn't the Jacobian weighting meant to deal with signal pileup in the _BOLD_ image? Would the field map type affect whether or not there is signal pileup due to distortion?

**Comment by effigies** [comment]
Yes, that's the idea, which is why I enabled it by default. In practice, it doesn't seem to work for all types, possibly because the empirical warps and the theoretical warps aren't matched closely enough.

**Comment by psadil** [comment]
Sorry to chime in, but was a least-squares restoration option considered? It's my impression that least-squares restoration rather than jacobian modulation is the default in standard fsl for PEPolar/`topup` (based on https://web.mit.edu/fsl_v5.0.10/fsl/doc/wiki/topup(2f)ApplytopupFurtherInformation.html)

**Comment by effigies** [comment]
That one only theoretically works for PEPolar fieldmaps. I don't see an obvious way to extend it to situations where we do not have two directions to work with. As to using it for PEPolar, it's an additional complication (you need to pull through the raw fieldmap inputs instead of just the preprocessed fieldmap), and the Jacobian approach seems to work pretty well already.
